devices/kobo-clara/firmware/default.nix

```diff
@@ -0,0 +1,9 @@
+{ runCommandNoCC
+}:
+
+runCommandNoCC "kobo-clara-firmware" {
+  src = ./epdc_PENG060D.fw;
```

Comment:
No documentation yet but this is a firmware file you'll have to extract from the factory SD card image.